### DISCUSSION TITLE
{code: 400, error_code: "validation_failed", msg: "Unsupported logout scope "window""}

### DISCUSSION THREAD

**Discussion by YA9** [supabase/supabase#27415] [Questions]
When I try supabase.auth.signOut(), on version 2.43.2 on JS, I get this error:
{
    "code": 400,
    "error_code": "validation_failed",
    "msg": "Unsupported logout scope \"window\""
}

Please help

**Comment by GaryAustin1** [comment]
You really need to provide more info on your call.
Signout() does have a scope as shown, but window is not a valid option.
![image](https://github.com/supabase/supabase/assets/54564956/34a73932-d226-432c-882b-c12d4ca67d45)

**Reply by YA9** [reply]
I didn't provide a scope. I think it's a bug displaying window. I just wrote the following:
```javascript
supabase.auth.signOut()
``` 
It works on localhost, but not when I build. I found a temporary workaround. If I define the scope as local, it works. By default I believe it's set to global, which is where the issue occurs.

**Reply by mkultracreative** [reply]
This is a major bug! Has been for over a year now. I get it on all frameworks.

**Reply by mkultracreative** [reply]
> I didn't provide a scope. I think it's a bug displaying window. I just wrote the following:
> 
> ```js
> supabase.auth.signOut()
> ```
> 
> It works on localhost, but not when I build. I found a temporary workaround. If I define the scope as local, it works. By default I believe it's set to global, which is where the issue occurs.

Can you provide the code you used at least for the local workaround? I’m so frustrated
